javascript-timers/interval.js

```diff
@@ -0,0 +1,14 @@
+
+const number = document.querySelector('.countdown-display');
+
+let counter = 4;
+function countdown() {
+  const boom = number.textContent = 'boom';
+  if (counter === -1) {
```

Comment:
You should be checking for greater than 0. We don't want to see 0, the text should replace 0